pkgs/development/python-modules/dotbot/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, fetchPypi
+, pytestCheckHook
+, pyyaml
+}:
+
+buildPythonPackage rec {
```

Comment:
Yeah, I think this is not meant to be used as a library, I'll update it

Should it still be placed in the `/pkg/development/python-modules/` path? I see that chezmoi (which does something similar) lives on `/pkg/tools/misc/`

Reply:
it can go in [pkgs/by-name/do/dotbot/package.nix](https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#name-based-package-directories) and the change to all-packages.nix can be reverted.

Reply:
in here https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#limitations it says that packages defined using `pkgs.python3Packages.callPackage` should not be migrated, is that right? does it apply for this package?

Reply:
the callPackage is not required. see the other python applications in `pkgs/by-name`. I think you'll have to add `python3Packages` to the arguments and then add it as a prefix for `python3Packages.buildPythonApplication`, `python3Packages.pyyaml`, setuptools and pytestHook. or just add `with python3Packages;` between the argument set and buildPythonApplication. I'm not sure which is better.

Reply:
oh I see, I'll apply that change